tools/cli/commands.go

```diff
@@ -383,13 +390,35 @@ func showHistoryHelper(c *cli.Context, wid, rid string) {
 	table.SetBorder(false)
 	table.SetColumnSeparator("")
 	for _, e := range history.Events {
+		columns := []string{}
+		if printFully {
+			jsonBs, err := json.Marshal(e)
+			var jsonStr string
+			if err != nil {
+				jsonStr = "ERROR!"
```

Comment:
Would be better to add a little bit more info here like "Error when marshal event" so that customers won't be confused whether it is server error or CLI error.

Reply:
Yeah, make sense